### PR TITLE
Add minor fixes and a small feature

### PR DESCRIPTION
**Pull request by jacobbloomCDD**
Fix missing files being triggered by empty files

Fix missing files also being skipped files

Add unique string option to prevent incorrect spec version execution

### CODE REVIEW

**Comment by jacobbloomCDD** [comment]
reasonably minor fixes

File: README.md [hunk 1]
```diff
@@ -101,6 +101,12 @@ A corresponding set up for a node node using SSH would be:
 
 N.B. Ensure that the head node has tcp local port forwarding permitted.
 
+A set of nodes and head node can be given a unique string in case multiple spec
+file versions may be tested near simultaneously to prevent execution of the
+wrong version.
+
+    $ multirspec -u SomeUniqueString
```
Comment (kdole) [inline README.md]: Maybe make it concrete for them: "(e.g. a git commit hash)".

As for naming, for some people the concept "build identifier" or "run identifier" might resonate more than "unique string", but up to you where you want to go with the terminology.

Comment (jacobbloomCDD) [inline README.md]: done

**Comment by kdole** [comment]
LGTM.

**Pull request merged**